src/Models/NonhydrostaticModels/show_nonhydrostatic_model.jl

```diff
@@ -1,12 +1,28 @@
 using Oceananigans.Utils: prettytime, ordered_dict_show
+using Oceananigans.TurbulenceClosures: closure_summary
 
-"""Show the innards of a `Model` in the REPL."""
-function Base.show(io::IO, model::NonhydrostaticModel{TS, C, A}) where {TS, C, A}
-    print(io, "NonhydrostaticModel{"*string(Base.nameof(A))*", $(eltype(model.grid))}",
-        "(time = $(prettytime(model.clock.time)), iteration = $(model.clock.iteration)) \n",
-        "├── grid: $(summary(model.grid))\n",
-        "├── tracers: $(tracernames(model.tracers))\n",
-        "├── closure: ", summary(model.closure), '\n',
+function Base.summary(model::NonhydrostaticModel)
```

Comment:
@glwagner : based on a conversation we had yesterday, it seems like what we have for Hydrostatic is almost identical to Nonhydrostatic.  Would it make sense to have a common `Base.summary` sometime, rather than having this repeated code in both?

Reply:
It would definitely be better to combine the implementations. `summary` wouldn't be hard. More important is using just one struct and constructor...

Reply:
Just wanted to mention the idea and could happen in a future PR.